library/src/morpheus-master/sass/base/_defaults.scss

```diff
@@ -213,3 +213,14 @@ ul, ol {
 span.marker {
     background-color: yellow;
 }
+
+.hide-input-arrows {
+  -moz-appearance: textfield;
+  &::-webkit-inner-spin-button,
+  &::-webkit-outer-spin-button {
+    -webkit-appearance: none;
+    -moz-appearance: none;
+    appearance: none;
```

Comment:
very good use case for a mixin!

https://github.com/drublic/Sass-Mixins/blob/master/partials/_appearance.scss

Reply:
What's the advantage to a mixin? Does it mean you can modify elements by intrumenting their existing classes? Ie: you don't need to go into the markup of the tool, you can just mod the compiled sass?

Reply:
it only makes sense if this pattern is being repeated .....  so instead of using all these -appearance attributes, it would be one call

if it's not a repeated pattern, ignore

Reply:
It's not at the moment. I moved it up out of rubrics as I was trying to use the number type in grader. That didn't work out, tbh, so I could just move it back down into the rubrics scss. It's not repeated, though, just a class you can layer on.

Reply:
I'm with @ottenhoff, I think this should be change to a mixin for using in the future

Reply:
For what reason, though? All you need to do is just add the class, bootstrap style, to your element, and you have the styles.

Reply:
If you need on other place another value for
-webkit-appearance: newvalue;
    -moz-appearance: newvalue;
    appearance: newvalue;

It's better if we have a mixin to parameterize this "newvalue" and only called
x-appearance like appears in https://github.com/drublic/Sass-Mixins/blob/master/partials/_appearance.scss

Reply:
That just seems like pre optimisation to me. Why bother if we don't have a use for it yet? But, if that pattern exists in other places then we should open another JIRA to clean up those places and consolidate in a mixin.